pages.es/common/adb-install.md

```diff
@@ -0,0 +1,28 @@
+# adb install
+
+> Instalación de Android Debug Bridge: Envía paquetes a una instancia del emulador de Android o a dispositivos Android conectados.
+> Más información: <https://developer.android.com/studio/command-line/adb>.
+
+- Envía una aplicación Android a un emulador/dispositivo:
+
+`adb install {{ruta/a/archivo.apk}}`
```

Comment:
```suggestion
`adb install {{ruta/al/archivo.apk}}`
```
Isn't it al? Now I am confused where you use a and al 😅

Reply:
it's gender based `el` for male terms, `la` for female terms. `a la` is `to the` (Spanish uses `the` more often than English).  but for `a el`, this is a contraction `al`, instead of `a el`.